### PR TITLE
Update convert-user-mailbox-to-shared-mailbox.md

### PR DESCRIPTION
**Pull request by dimcry**
Correcting the part of the article discussing about converting an user mailbox to a shared mailbox in an Exchange Hybrid environment.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 1]
```diff
@@ -88,13 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
...
-
-Usually this is not a problem, but there are some scenarios where the on-premises attributes (which think that the mailbox is the user mailbox) can overwrite the new cloud versions of those attributes, and as a result, the mailbox might convert back. This is a problem because user mailboxes require licenses **or they are soft deleted after 30 days**!
-
-We've addressed most of the reasons why this happens but it still CAN happen, although infrequently. It's best to be safe and move the mailbox back to on-premises, convert it, and then move the shared mailbox back to the cloud. This recommended solution is not in violation of the licensing agreement for hybrid environments because the existence of the user mailbox on-premises is only temporary. You would be in violation of your license if you maintained the user mailbox or shared mailbox in your on-premises organization and did not move it back to the cloud.
+More details about converting an user mailbox to a shared mailbox, into an Exchange Hybrid environment can be found on:
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:91]:
```suggestion
For details about converting a user mailbox to a shared mailbox in an Exchange hybrid environment, see [Cmdlets to create or modify a remote shared mailbox in an on-premises Exchange environment](https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49) and [Shared mailboxes are unexpectedly converted to user mailboxes after directory synchronization runs in an Exchange hybrid deployment](https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes).
```

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 2]
```diff
@@ -88,13 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
...
+https://support.microsoft.com/office/cmdlets-to-create-or-modify-a-remote-shared-mailbox-in-an-on-premises-exchange-environment-9e83fb59-c001-729c-a4c0-b2964c154b49
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:92]:
```suggestion
```

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md [hunk 3]
```diff
@@ -88,13 +88,9 @@ Let's say you've deleted a user account and now you want to convert their old ma
...
+https://docs.microsoft.com/exchange/troubleshoot/user-and-shared-mailboxes/shared-mailboxes-unexpectedly-converted-to-user-mailboxes
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:93]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by dimcry** [comment]
@JohanFreelancer9 is something that I should continue / do, in order to have this change in Production?

**Comment by JohanFreelancer9** [comment]
@dimcry - yes, please merge the commit on the copy/edit review I've done. Thank you.

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md (hunk 1 above)
Comment (dimcry) [reply microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:91]: Thank you!

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md (hunk 2 above)
Comment (dimcry) [reply microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:92]: Thank you!

File: microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md (hunk 3 above)
Comment (dimcry) [reply microsoft-365/admin/email/convert-user-mailbox-to-shared-mailbox.md:93]: Thank you!

**Comment by dimcry** [comment]
Merging the content in Production.

**Comment by dimcry** [comment]
Seems that I don't have rights to commit...

**Comment by dimcry** [comment]
@JohanFreelancer9 / @yogkumgit seems that I don't have rights to merge this in Production.
Can you please help me with this?

Thank you very much!

**Comment by yogkumgit** [comment]
@dimcry and @JohanFreelancer9  confirm copy review

**Comment by JohanFreelancer9** [comment]
@yogkumgit - all good after my edits have been implemented.

**Comment by dimcry** [comment]
@yogkumgit I confirm it can be published. Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SKjerland Please merge this PR. Thanks.

**Comment by SKjerland** [comment]
@yogkumgit or @JohanFreelancer9 - Please resolve the conflicts and then ping me again so I can merge. Thanks!

**Comment by JohanFreelancer9** [comment]
@yogkumgit - unfortunately I cannot resolve the conflicts, can you please assist?

**Comment by yogkumgit** [comment]
@msbemba Please help and resolve this conflict if possible. Thanks

**Comment by msbemba** [comment]
@yogkumgit I don't have access to resolve this conflict. Only those who created the PR can resolve the conflict.

**Comment by SKjerland** [comment]
@dimcry - Can you please resolve the conflicts on this PR so that I can merge? Thanks!

**Comment by dimcry** [comment]
@yogkumgit / @msbemba  / @SKjerland / @JohanFreelancer9 please review and proceed in merging this change in Prod. Thank you!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SKjerland

**Pull request merged**